src/cpp/src/tokenizer.cpp

```diff
@@ -91,11 +96,25 @@ class Tokenizer::TokenizerImpl {
         // Try to read tokenizer_config if some token ids or token str are not defined.
         read_tokenizer_config_if_necessary(tokenizer_path); 
 
+        std::map<std::string, ov::Any> plugin_config{{"PERFORMANCE_HINT", "THROUGHPUT"}};
```

Comment:
My expectation was that tokenizers are single threaded. @apaniukov, does this affect anything?

Reply:
Yes, the throughput hint massively increases the async inference scenario. For sync inference, a latency hint is better.

Reply:
That means synchronous scenario becomes slower with that change?

Reply:
Yes, the average latency is a bit bigger in that scenario. There is a benchmarking script that measures sync and async inference stats for a particular tokenizer: https://github.com/openvinotoolkit/openvino_tokenizers/tree/master/benchmark
You can pass `--tput` flag to activate the throughput hint for the benchmark.

Reply:
To enable latency and tput scenarios I have added tokenizer plugin config as constructor argument. This way existing workflows will not be modified, but in OVMS we will manually set tput mode.

Please re-review.